desktop/src/renderer/src/pages/transfers/LanTransfer.tsx

```diff
@@ -517,10 +517,16 @@ export const LanTransferPage = () => {
 
   useEffect(() => {
     if (!fileList) {
+      // Clear state.
+      setAccession(null);
+      setApplication(null);
+      setConfirmAccAppChecked(false);
       setFolders([]);
       setMetadata({});
+      return;
     }
-    parseFileList();
+    setProgressMade(!!fileList); // Track if progress is made.
```

Comment:
Better just use `setProgressMade(true)`?
If fileList is `undefined` or `null`, it won't make it to this line anyhow
Also this method is to "set progress made", not tracking, the comment is a little unnecessary
```suggestion
    setProgressMade(true); 
```

Reply:
Actually it should go before the if statement because we also want to setProgressMade to false when file list is removed. I will update